_posts/2018-12-03-how-to-make-golden-egg-fried-rice.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "How To Make Golden Fried Rice"
+date: 2018-12-03 21:20:12
+tags:
+- Food
+- chinese cuisine
+categories: food recipes
+twitter_text: "How To Make Golden Fried Rice"
+authors: Cheng Yang
+image: https://c1.staticflickr.com/3/2466/3708994257_4e359233c9_b.jpg
+image_url: https://www.flickr.com/photos/35856236@N06/3708994257/
+image_credit: hawaiianfoodrecipe
+---
+
+Egg Fried Rice is a very homely dish in China. Every Chinese family make their fried rice differently, so you don't actually need a receipt to make fried rice, you can add whatever you like into the rice. Today, I am going to show you my method to make this simple, quick and delicious dish.
```

Comment:
`receipt` should be `recipe`.

I think that `you can add whatever you like into the rice` should be something more like `you can add whatever you want to the dish`

 `simple, quick and delicious dish.` should be `simple, quick, and delicious dish.`

Reply:
Thanks :D